ff/serving/provider/postgres.go

```diff
@@ -590,16 +747,270 @@ func (iter *postgresFeatureIterator) Err() error {
 
 // castTableItemType returns the value casted as its original type
 func castPostgresTableItemType(v interface{}, t postgresColumnType) interface{} {
+	if v == nil {
+		return v
+	}
 	switch t {
 	case PGInt:
 		return int(v.(int32))
+	case "bigint":
+		return int(v.(int64))
 	case PGFloat:
 		return v.(float64)
 	case PGString:
 		return v.(string)
 	case PGBool:
 		return v.(bool)
+	case PGTimestamp:
+		return v.(time.Time).UTC()
 	default:
 		return v
 	}
 }
+
+// CreateTransformation starts a transformation query that saves into the specified output table.
+// If a generic query is specified, the output is stored back into a new Primary Table.
+// If a Feature or Label query is specified, it stores the output into a Feature or Label
+// Resource table and maps the columns to fit the Label or Feature table schema. It also
+// adds a unique constraint to make it possible to materialize the Label or Feature.
+func (store *postgresOfflineStore) CreateTransformation(config TransformationConfig) error {
+	name, err := store.createTransformationName(config.TargetTableID)
+	if err != nil {
+		return err
+	}
+	// Only allow transformations to be run with SELECT queries
+	splitQuery := strings.Split(config.Query, " ")
```

Comment:
I do want to figure out how to do this, but there are situations where SELECt isn't the first word. Let's just assume its a select for now and let the SQL query fail on the backend